fake-koji/src/main/java/org/fakekoji/xmlrpc/server/core/FakeBuild.java

```diff
@@ -375,30 +368,20 @@ private File getNewestFile() {
     private static final String[] defaultSupportedArchs = new String[]{"x86_64", "i686", "win"/*, "aarch64"*/};
     //aarch64 is now added in rnutime, via  per project  settings. When ojdk9 will be oldest suported jdk, it have sense to put it here.
 
-    private String[] getSupportedArches() throws IOException {
+    private String[] getSupportedArches() {
         ServerLogger.log("For: " + getNVR());
-        if (getArchesConfigFile().exists()) {
-            ServerLogger.log("Using expected arches from " + getArchesConfigFile().getAbsolutePath());
-            return readArchesFile(getArchesConfigFile());
-        }
-        if (getProjectConfigFile1().exists()) {
-            ServerLogger.log("Using expected arches from " + getProjectConfigFile1().getAbsolutePath());
-            return readArchesFile(getProjectConfigFile1());
-        }
-        if (getProjectConfigFile2().exists()) {
-            ServerLogger.log("Using expected arches from " + getProjectConfigFile2().getAbsolutePath());
-            return readArchesFile(getProjectConfigFile2());
+        try {
+            List<String> arches = this.projectMapping.getExpectedArchesOfNVR(getNVR());
+            ServerLogger.log("Using project default expected arches");
+            return arches.toArray(new String[0]);
+        } catch (ProjectMappingExceptions.ProjectMappingException e) {
+            ServerLogger.log("Using hardcoded arches.");
+            return defaultSupportedArchs;
```

Comment:
hmhmh. Cant this return nothing? If it have to retuern something, then the constant shold be kept. However without comments, and without comented out values.